### ISSUE TITLE
Support for Redmine behind HTTP Auth.

### ISSUE THREAD

**Issue by jamesepatrick** [jason0x43/alfred-redmine#5]
Hey Jason0x43,

I haven't been able to find a way to connect to a Redmine Instance that is behind Basic HTTP Auth. I was wondering if there way any undocumented way to do this. I tried to connect using `user:pass@example.com` but it errors out when with a message saying that it was unable to connect to the host. 

If this isn't currently supported and you are open to adding this, I'd be willing to add this and sent a pull request.

**Comment by jason0x43** [comment]
Just to clarify, you're interested in Basic Auth that's separate from Redmine's own Basic Auth (which is just an alternative to using an API key)?

**Comment by jamesepatrick** [comment]
Yep. The HTTP Basic Auth, not Redmine's Basic Auth.

**Comment by jason0x43** [comment]
Hmmm...ok. Redmine supports HTTP Basic Auth, and there's some allowance for using it in the workflow, but just enough to figure out the user's API key (because ideally the workflow shouldn't store a username and password). I'll need to add something to support a completely separate set of basic auth credentials, and preferably to store them in the keychain.

**Comment by jamesepatrick** [comment]
Hey Jason. I meant as in a password protected reverse proxy using a web server (eg Apache).

**Comment by jason0x43** [comment]
Yeah, that's why it needs to support separate (non-Redmine) credentials.

**Comment by jason0x43** [comment]
Totally doable.
